packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4944,6 +4972,48 @@ export default class Meeting extends StatelessWebexPlugin {
     }
   }
 
+  /**
+   * This method will toggle the highlights for the current meeting if the meeting has enabled/supports Webex Assistant
+   * @param {Boolean} activate flag to enable/disable highlights
+   * @param {Object} options object with spokenlanguage setting
+   * @public
+   * @returns {Promise<void>} a promise to open the WebSocket connection
+   */
+  public async toggleHighlighting(activate: boolean, options?: {spokenLanguage?: string}) {
+    if (this.isJoined() && this.isTranscriptionSupported()) {
+      LoggerProxy.logger.info(
+        'Meeting:index#toggleHighlighting --> Attempting to enable highlights!'
+      );
+
+      try {
+        if (activate) {
+          // @ts-ignore
+          this.webex.internal.voicea.on(
+            VOICEAEVENTS.HIGHLIGHT_CREATED,
+            this.voiceaListenerCallbacks[VOICEAEVENTS.HIGHLIGHT_CREATED]
+          );
+        } else {
+          // @ts-ignore
+          this.webex.internal.voicea.off(
+            VOICEAEVENTS.HIGHLIGHT_CREATED,
+            this.voiceaListenerCallbacks[VOICEAEVENTS.HIGHLIGHT_CREATED]
+          );
+        }
+        // @ts-ignore
+        await this.webex.internal.voicea.toggleTranscribing(activate, options?.spokenLanguage);
+      } catch (error) {
+        LoggerProxy.logger.error(`Meeting:index#toggleHighlighting --> ${error}`);
+        Metrics.sendBehavioralMetric(BEHAVIORAL_METRICS.RECEIVE_TRANSCRIPTION_FAILURE, {
+          correlation_id: this.correlationId,
+          reason: error.message,
+          stack: error.stack,
+        });
+      }
+    } else {
+      throw new Error('This operation is not allowed in your org. Contact org administrator.');
```

Comment:
We throw this error if we find that transcriptions are not enabled. Usually happens if the old webex assistant is not enabled for the org.